app/javascript/components/Study.js

```diff
@@ -93,7 +93,15 @@ function facetMatchBadges(study) {
           className="badge badge-secondary facet-match"
           data-toggle="tooltip"
           title={helpText}>
-          { matches[key].map(filter => filter.name).join(',') }
+          {
+            matches[key].map(filter => {
+              if (filter.min) { // numeric facet
+                return `${key} ${filter.min}-${filter.max} ${filter.unit}`
```

Comment:
The screenshot shows "organism_age: 1-180 years".  Instead of showing the ID for such matches, we should its name, e.g. "Organism Age: 1-180 years":

```suggestion
                return `${filter.name} ${filter.min}-${filter.max} ${filter.unit}`
```

Reply:
yeah, the backend doesn't return the display name in these matches, so this was me going to quick-and-dirty route.  I think you're right though that organism_age is too sloppy.  So I'll get the display name via other means

Reply:
The `name` attribute is source from Mongo, so if the value you're getting back is not what we want, we should change it in the database.

Reply:
the backend only returns the 'name' for filters, not for facets.  And the numeric slider badge is different from the others in that we need the name of the facet, not the filter